torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -481,7 +459,8 @@ c10::intrusive_ptr<JitFuture> RequestCallbackImpl::processRpcWithErrors(
                    // recorded the exception in the response message.
     return future;
   } catch (std::exception& e) {
-    return asFuture(handleError(e, messageType, messageId));
+    // Pass a dummy message ID since it will be overwritten anyways.
```

Comment:
should we remove message id from  `handleError` argument list?

Reply:
Please feel free to do this in followup PRs.

Reply:
My long-term idea is to remove the ID entirely from Message. I believe the ID is only used by the RPC agent to match requests with responses, so in a sense it's an "external" information (it shouldn't be _inside_ the Message, it should be a "tag" that's attached to it).